include/libdnf5/offline/offline.hpp

```diff
@@ -0,0 +1,95 @@
+/*
```

Comment:
Are there any plans or considerations for utilizing the `libdnf5::offline` module within the library? Currently, it seems to be associated only with reading/writing offline transaction states, processed only in the `dnf5::Context`, is that correct? Maybe we can consider moving it to the dnf5 part?

Reply:
Good thought, I've moved it out of libdnf5 to `dnf5/include/dnf5/offline.hpp`.